src/odemis/driver/xt_client.py

```diff
@@ -350,273 +349,269 @@ def check_and_transfer_latest_package(self) -> None:
                     # Notify the user that a newer xtadpater version is available
                     notify2.init("Odemis")
                     update = notify2.Notification(
-                        "Update Delmic XT Adapter", "Newer version {} is available on ThermoFisher Support PC.\n\n"
-                        "How to update?\n\n1. Full stop Odemis and close Delmic XT Adapter.\n2. Restart the Delmic XT Adapter "
-                        "to install it.".format(package.version))
+                        "Update Delmic XT Adapter",
+                        "Newer version {} is available on ThermoFisher Support PC.\n\n"
+                        "How to update?\n\n1. Full stop Odemis and close Delmic XT Adapter.\n"
+                        "2. Restart the Delmic XT Adapter to install it.".format(package.version))
                     update.set_urgency(notify2.URGENCY_NORMAL)
-                    update.set_timeout(10000)    # 10 seconds
+                    update.set_timeout(10000)  # 10 seconds
                     update.show()
                 else:
                     logging.warning("{} is a bad file in {} not transferring latest package.".format(ret, package.path))
         except Exception as err:
             logging.exception(err)
 
-    def list_available_channels(self):
+    def list_available_channels(self) -> dict:
```

Comment:
I just noticed that this function is not there anymore in the xtadapter, so I will completely remove it. Should I do that as a separate commit in this PR or in a new one?

Reply:
There are actually no user? If so, the removal can be very simple, and just as a separate commit of this PR. Otherwise, just keep it for later, as a separate PR.

Reply:
There is no actual user, I'll then do it as a separate commit on this PR.